packages/top-app-bar-fixed/README.md

```diff
@@ -0,0 +1,138 @@
+
+> IMPORTANT: The Material Web Components are a work in progress and subject to
+> major changes until 1.0 release.
+
+Fixed Top App Bars are a container for items such as application title, navigation icon, and action items that are always visible.
+
+![](images/fixed.gif)
+
+For a version of this component that scrolls, see [`<mwc-top-app-bar>`](http://github.com/material-components/material-components-web-components/tree/master/packages/top-app-bar)
+
+For a collapsable version of this component, see [`<mwc-top-app-bar-short>`](http://github.com/material-components/material-components-web-components/tree/master/packages/top-app-bar-short)
+
+[Material Design Guidelines: App Bars: Top](https://material.io/design/components/app-bars-top.html)
+
+## Installation
+
+```sh
+npm install @material/mwc-top-app-bar-fixed
+```
+
+> NOTE: The Material Web Components are distributed as ES2017 JavaScript
+> Modules, and use the Custom Elements API. They are compatible with all modern
+> browsers including Chrome, Firefox, Safari, Edge, and IE11, but an additional
+> tooling step is required to resolve *bare module specifiers*, as well as
+> transpilation and polyfills for Edge and IE11. See
+> [here](https://github.com/material-components/material-components-web-components#quick-start)
+> for detailed instructions.
+
+## Example Usage
+
+### Standard
+
+<img src="images/standard.png" height="56px">
+
+```html
+<mwc-top-app-bar-fixed>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Dense
+
+<img src="images/dense.png" height="48px">
+
+```html
+<mwc-top-app-bar-fixed dense>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Prominent
+
+<img src="images/prominent.png" height="128px">
+
+```html
+<mwc-top-app-bar-fixed prominent>
+    <mwc-icon-button icon="menu" slot="navigationIcon"></mwc-icon-button>
+    <div slot="title">Title</div>
+    <mwc-icon-button icon="file_download" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="print" slot="actionItems"></mwc-icon-button>
+    <mwc-icon-button icon="favorite" slot="actionItems"></mwc-icon-button>
+    <div><!-- content --></div>
+</mwc-top-app-bar-fixed>
+```
+
+### Prominent and Dense
+
+<img src="images/prominent_and_dense.png" height="96px">
```

Comment:
The title cuts off in this screenshot.
<img src="https://user-images.githubusercontent.com/48894/63533529-cf782300-c4c1-11e9-98b5-d4c2abbd9e65.png" width="50%">

Reply:
I think I can remove the `Fixed` part from these, and that should fix it.

Reply:
Done